### PR TITLE
Add ES6 Support

### PR DESCRIPTION
**Pull request by synbot**
### Acceptance Criteria
1. Application builds without errors

### Tasks
- Add babel & babel loaders
- Enable esnext so linter doesn't complain

### CODE REVIEW

**Comment by kylecesmat** [comment]
:warn: waiting for #120

File: application/bootstrap.js [hunk 1]
```diff
@@ -64,6 +64,6 @@ router.run(function (Handler, state) {
             locales  : locales,
             messages : i18n.messages
         }),
-        window.document.body
+        window.document.getElementById('app')
```
Comment (paulstatezny) [inline application/bootstrap.js]: Why this change/

Comment (kylecesmat) [inline application/bootstrap.js]: this was on #120

Comment (paulstatezny) [inline application/bootstrap.js]: Ohh, so is this blocked by #120?

**Comment by paulstatezny** [comment]
Looks good from a dev standpoint. Just waiting on #120 to be merged before I can star2.

**Comment by kylecesmat** [comment]
:o:

**Comment by kylecesmat** [comment]
:star:

**Comment by paulstatezny** [comment]
:pray:

**Comment by paulstatezny** [comment]
To test this, just make sure the app runs as normal

**Comment by aecarr2** [comment]
:-1: Error in terminal, not spinning up.

**Comment by aecarr2** [comment]
```ERROR in ./application/ui/scss/pattern-library.scss
Module build failed: ModuleBuildError: Module build failed: 
    @media #{$large-up} {
          ^
      Undefined variable: "$large-up".
      in /Users/austincarr/Documents/My Documents/GitHub/frontend-template/application/ui/scss/pattern-library/_pl-layout.scss (line 39, column 12)
    at DependenciesBlock.onModuleBuildFailed (/Users/austincarr/Documents/My Documents/GitHub/frontend-template/node_modules/webpack/node_modules/webpack-core/lib/NormalModuleMixin.js:315:19)
    at nextLoader (/Users/austincarr/Documents/My Documents/GitHub/frontend-template/node_modules/webpack/node_modules/webpack-core/lib/NormalModuleMixin.js:270:31)
    at /Users/austincarr/Documents/My Documents/GitHub/frontend-template/node_modules/webpack/node_modules/webpack-core/lib/NormalModuleMixin.js:292:15
    at context.callback (/Users/austincarr/Documents/My Documents/GitHub/frontend-template/node_modules/webpack/node_modules/webpack-core/lib/NormalModuleMixin.js:148:14)
    at Object.onRender (/Users/austincarr/Documents/My Documents/GitHub/frontend-template/node_modules/sass-loader/index.js:152:13)
    at options.error (/Users/austincarr/Documents/My Documents/GitHub/frontend-template/node_modules/node-sass/lib/index.js:279:32)
 @ ./application/ui/pages/pattern-library.jsx 13:0-34
Child extract-text-webpack-plugin:
        + 1 hidden modules
    
    ERROR in ./~/css-loader!./~/postcss-loader!./~/sass-loader?outputStyle=nested&includePaths[]=/Users/austincarr/Documents/My Documents/GitHub/frontend-template/node_modules!./application/ui/scss/pattern-library.scss
    Module build failed: 
        @media #{$large-up} {
              ^
          Undefined variable: "$large-up".
          in /Users/austincarr/Documents/My Documents/GitHub/frontend-template/application/ui/scss/pattern-library/_pl-layout.scss (line 39, column 12)```

**Comment by kylecesmat** [comment]
:pray: Broke when i refreshed it due to #121

**Comment by kylecesmat** [comment]
:star2:

**Comment by nearengine** [comment]
:+1:
